goreleaser.yml

```
@@ -0,0 +1,19 @@
+project_name: pillarbox-event-dispatcher
```

Maybe the name could be replaced in this file too?
```suggestion
project_name: pillarbox-monitoring-dispatch
```